core/fx.d.ts

```
@@ -0,0 +1,489 @@
+    __parent_id: string | null;
+    __nodes: Record<string, FXNode>;
+    __value: {
+      str?: string;
+      num?: number;
+      bool?: boolean;
+      json?: object;
+      [key: string]: any;
+    };
+    __type: string | null;
+    __proto: string[];
+    __effects: Function[];
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)